### ISSUE TITLE
Add more visualization. 

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/js_intro_jquery#18]
See conversation on https://github.com/ga-wdi-boston/jquery-dom/pull/16

**Comment by RealWeeks** [comment]
@gaand can you add more detail. Looks like #16 was closed then reopened for discussion and is pending comment. Do you mean images in the repo or activities to help devs visualize. There are diagraming labs, should there be more?

**Comment by payne-chris-r** [comment]
I've added a quick "Draw-along" where the class draws the same DOM diagram that I do and we talk about bubbling using it. Ok to close @gaand ?

**Comment by payne-chris-r** [comment]
Would actually love an idea of what to name that--It's not a code-along, and it's not a lab. Follow along? Draw-along? Should we just name it 'code-along' to be consistent?

**Comment by gaand** [comment]
I think that was part of what I saw happening.  An important part is to relate the structure of the html to the structure of the DOM, but to make it clear that they are not the same thing.]

I suggest a lab of the squads given html and drawing a DOM for it.

**Comment by raq929** [comment]
@gaand Has this been addressed to your satisfaction, or is there more that I should add.

**Comment by gaand** [comment]
I'd like to see a bit more in these sections

https://github.com/ga-wdi-boston/jquery-dom#demo-diagram-the-dom
https://github.com/ga-wdi-boston/jquery-dom#lab-diagram-the-dom

Specifically, relating a wireframe to html to the DOM.

This can wait for the next iteration.

**Comment by raq929** [comment]
I agree that those sections need more. They haven't had the wireframes lesson yet, so maybe some material could be added to https://github.com/ga-wdi-boston/project-planning-wireframes to relate wireframe - html - dom?

**Comment by raq929** [comment]
Should that go here, or in the wireframes lesson? Tying them together in
that lesson might be nice. (Will link to lesson when I'm not on my phone.)
